### DISCUSSION TITLE
setting up MOOSE

### DISCUSSION THREAD

**Discussion by MahshadFani** [idaholab/moose#26908] [Q&A Getting Started]
Dear all:

When I wanted to test the MOOSE application the following error occurred, I repeated it some times but still showed these errors.

Could you please help me fix these errors? Thank you!

[Terminal Saved Output.txt](https://github.com/idaholab/moose/files/14426202/Terminal.Saved.Output.txt)

**Comment by GiudGiud** [comment]
Hello

the wasp package is too old
moose-wasp public/osx-arm64::moose-wasp-2023.11.29-build_0

We re on this one now
moose-wasp 2024.02.19 build_0 https://conda.software.inl.gov/public

so if you could try to update it specifically with conda. If it does not work, then delete the moose environment on conda, and re-install it

Guillaume
